### PR TITLE
Family missions widget

### PR DESCRIPTION
**Pull request by MarySnopok**
Texts for activation screen and string key change to meed new usage

### CODE REVIEW

File: text_strings/client/en.json [hunk 1]
```diff
@@ -1306,8 +1306,8 @@
   "general_do_again": "Do again",
   "general_locked": "Locked",
   "kid_profile_tab_adventures": "Missions",
-  "flash_card_family_missions_title": "Family missions",
-  "flash_card_family_missions_desc": "Go through the basics of personal finances together with your child!",
+  "widget_title_family_missions": "Family missions",
+  "widget_description_family_missions": "Go through the basics of personal finances together with your child!",
```
Comment (gimi-anders) [inline text_strings/client/en.json:1310]: Why changing the name of these keys? Aren't these flash cards showing any more?
Also, if the previous key names are in use, then we cannot merge this PR until we merge a `gimi-app` PR that fixes it.

**Review by gimi-anders — CHANGES_REQUESTED** [review]

File: text_strings/client/en.json (hunk 1 above)
Comment (MarySnopok) [reply text_strings/client/en.json:1310]: @gimi-anders I renamed it cause it was used only within one card kidProfileParentFamilyMissions , which was an entry point for family missions and used only once. Now when we delete the Missions tab and family missions card we can/should rename the strings . I searched for these strings and flash card within repo, cant find any other usage. You can double check to be 100% sure.

Comment (gimi-anders) [reply text_strings/client/en.json:1310]: The issue with renaming though is that we cannot merge this PR until we certainly know that the usage is removed/changed in the app.
For example, if I would have merged this PR yesterday, and today we make the release, obviously without changing the family missions code in the app. Then it means that the flash card would be missing the string.

Comment (gimi-anders) [reply text_strings/client/en.json:1310]: Anyway, the release is branched out now, so it should be safe to merge this as long as the app code is updated before next release.

**Review by gimi-anders — APPROVED** [review]

**Comment by MarySnopok** [comment]
@gimi-anders  Anyway I have conflicts to resolve. I can also revert back the old string and we will be on safe side. If it wont be used in future, then your clean up script will remove it. I will take it back

**Review by gimi-anders — APPROVED** [review]

**Pull request merged**